hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -411,7 +411,7 @@ private FlinkOptions() {
   public static final ConfigOption<String> RECORD_KEY_FIELD = ConfigOptions
       .key(KeyGeneratorOptions.RECORDKEY_FIELD_NAME.key())
       .stringType()
-      .defaultValue("uuid")
+      .defaultValue("")
```

Comment:
noDefaultValue?

Reply:
Hi , KeyGeneratorOptions.RECORDKEY_FIELD_NAME's default value is noDefaultValue(), I guess we can also change .defaultValue("")  to  .noDefaultValue().

Reply:
Yeah, but make sure to go through the validations in `HoodieTableFactory` to keep the validity.

Reply:
You are right, I will test the several situations, and then consider it is necessary or not to change.